coverage/index.html

```diff
@@ -64,7 +64,7 @@ <h1>
 </div><!-- /wrapper -->
 <div class='footer quiet pad2 space-top1 center small'>
   Code coverage
-  generated by <a href="http://istanbul-js.org/" target="_blank">istanbul</a> at Tue Jul 24 2018 17:44:20 GMT-0400 (EDT)
+  generated by <a href="http://istanbul-js.org/" target="_blank">istanbul</a> at Fri Jul 27 2018 16:37:23 GMT-0400 (EDT)
```

Comment:
coverage should be omitted in your repository. Check staff announcements for a post from me on how to mitigate the coverage directory from appearing in source control.